OSDOCS#14731: Update the z-stream RNs for 4.18.14 (#93584)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.18.14
[id="ocp-4-18-14_{context}"]
=== RHSA-2025:7863 - {product-title} {product-version}.14 bug fix update and security update

Issued: 20 May 2025

> {product-title} release {product-version}.14 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7863[RHSA-2025:7863] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:7865[RHBA-2025:7865] advisory.

Reviewer: FYI, the second link doesn't work yet.

Author: @lahinson the 'rpm' advisory is in the 'REL_PREP' stage. It will be in the 'SHIPPED_LIVE' stage sometime this afternoon, and the advisory will be available at that time. Thanks!

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.18.14 --pullspecs
----

[id="ocp-4-18-14-bug-fixes_{context}"]
==== Bug fixes

> * Previously, the omission of the `OLMManagedLabelKey` label on objects resulted in cluster operation failures. With this release,  an update improves pod stability and ensures that the Operator Lifecycle Manager operates properly. (link:https://issues.redhat.com/browse/OCPBUGS-56098[OCPBUGS-56098])

Reviewer: ```suggestion
* Previously, the omission of the `OLMManagedLabelKey` label on objects resulted in cluster operation failures. With this release, an update improves pod stability and ensures that the Operator Lifecycle Manager operates properly. (link:https://issues.redhat.com/browse/OCPBUGS-56098[*OCPBUGS-56098*])
```

> * Previously, an invalid tar extraction format resulted in an improper file separation in the `ramdisk` logs, and caused file separators to appear randomly. With this release, an updated `ramdisk` log file  processes tar entries individually. This fix improves log readability, making them easier to interpret. (link:https://issues.redhat.com/browse/OCPBUGS-55938[OCPBUGS-55938])

Reviewer: ```suggestion
* Previously, an invalid `.tar` extraction format resulted in an improper file separation in the `ramdisk` logs, and caused file separators to appear randomly. With this release, an updated `ramdisk` log file  processes `.tar` entries individually. This fix improves log readability, making them easier to interpret. (link:https://issues.redhat.com/browse/OCPBUGS-55938[*OCPBUGS-55938*])
```
ISG: https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories

* Previously, incorrectly formatted proxy variables in an external binary resulted in build failures. With this release, an update removes proxy environment variables from the build pod and prevents any build failures. (link:https://issues.redhat.com/browse/OCPBUGS-55699[OCPBUGS-55699])

> * Previously, there was no event logged when an error occurred from failed conversion from ingress to route conversion. With this update, this error appear in the event logs. (link:https://issues.redhat.com/browse/OCPBUGS-55338[*OCPBUGS-55338*])

Reviewer: ```suggestion
* Previously, no event was logged when an error occurred from failed conversion from ingress to route. With this update, this error appear in the event logs. (link:https://issues.redhat.com/browse/OCPBUGS-55338[*OCPBUGS-55338*])
```

> * Previously, a missing `afterburn` package resulted in the failure of the `gcp-hostname.service`, which causied the `scale-up` job to fail, impacting end user deployments. With this release, the `afterburn` package is installed in the RHEL `scale-up` job. This fix enables a successful `scale-up` action, resolving the `gcp-hostname` service failure. (link:https://issues.redhat.com/browse/OCPBUGS-55158[*OCPBUGS-55158*])

Reviewer: ```suggestion
* Previously, a missing `afterburn` package resulted in the failure of the `gcp-hostname.service`, which caused the `scale-up` job to fail, impacting end-user deployments. With this release, the `afterburn` package is installed in the {op-system-base} `scale-up` job. This fix enables a successful `scale-up` action, resolving the `gcp-hostname` service failure. (link:https://issues.redhat.com/browse/OCPBUGS-55158[*OCPBUGS-55158*])
```

> * Previously, there was no communication between a `localnet` pod and a pod in the default network when both pods are on the same node. With this release, an update fixes this problem when the endpoints are on a different node. (link:https://issues.redhat.com/browse/OCPBUGS-55016[*OCPBUGS-55016*])

Reviewer: ```suggestion
* Previously, there was no communication between a `localnet` pod and a pod in the default network when both pods were on the same node. With this release, an update fixes this problem when the endpoints are on a different node. (link:https://issues.redhat.com/browse/OCPBUGS-55016[*OCPBUGS-55016*])
```
I'm a little confused by this bug-fix description. The first sentence sounds like the problem happens when both pods are on the same node. The second sentence sounds like the problem is fixed when endpoints are on different nodes. Is that correct?

Author: Thanks @lahinson, I updated the bug text and fixed the wording.

> * Previously, image pull timeouts occurred due to `Zscaler` scanning all data transfers. This resulted in timed out image pulls. With this release, the image pull timeout is increased to 30 seconds, allowing successful updates. (link:https://issues.redhat.com/browse/OCPBUGS-54663[*OCPBUGS-54663*])

Reviewer: If possible, add a noun after `Zscaler` to indicate what it is.

* Previously, you could add white space to {aws-first} tag names, but the installation program did not support them. This situation resulted in the installation program returning an `ERROR failed to fetch Metadata` message. With this release, the regular expression for {aws-short} tags now validates any tag name that has white space. The installation program accepts these tags and no longer returns an error because of white space. (link:https://issues.redhat.com/browse/OCPBUGS-53221[*OCPBUGS-53221*])

> * Previously, cluster nodes repeatedly lost communication due to improper remote port binding by Open Virtual Network (OVN)-Kubernetes. This affected pod communication across nodes. With this release, the remote port binding functionality is updated to be handled by OVN directly, improving the reliability of cluster node communication. (link:https://issues.redhat.com/browse/OCPBUGS-51144[OCPBUGS-51144])

Reviewer: ```suggestion
* Previously, cluster nodes repeatedly lost communication due to improper remote port binding by Open Virtual Network (OVN)-Kubernetes. This affected pod communication across nodes. With this release, the remote port binding functionality is updated to be handled by OVN directly, improving the reliability of cluster node communication. (link:https://issues.redhat.com/browse/OCPBUGS-51144[*OCPBUGS-51144*])
```

Author: @lahinson We are removing the bold effect for the Jiras. For some releases, we haven't gone through removing all of the asciidoc formatting from all of the releases, but working on it...thanks!

Reviewer: @tmalove Good to know -- thanks! I wasn't sure why some of the Jiras were bold but others weren't.
